Modelica/Thermal/HeatTransfer/UsersGuide/package.mo

```diff
@@ -0,0 +1,73 @@
+within Modelica.Thermal.HeatTransfer;
+package UsersGuide "User's Guide"
+  extends Modelica.Icons.Information;
+
+
+
+
+  annotation (
+    preferredView="info",
+    DocumentationClass=true,
+    Documentation(info="<html>
+<p>
+This package contains components to model <strong>1-dimensional heat transfer</strong>
+with lumped elements. This allows especially to model heat transfer in
+machines provided the parameters of the lumped elements, such as
+the heat capacity of a part, can be determined by measurements
+(due to the complex geometries and many materials used in machines,
+calculating the lumped element parameters from some basic analytic
+formulas is usually not possible).
+</p>
+<p>
+Example models how to use this library are given in subpackage <strong>Examples</strong>.<br>
+For a first simple example, see <strong>Examples.TwoMasses</strong> where two masses
+with different initial temperatures are getting in contact to each
+other and arriving after some time at a common temperature.<br>
+<strong>Examples.ControlledTemperature</strong> shows how to hold a temperature
+within desired limits by switching on and off an electric resistor.<br>
+A more realistic example is provided in <strong>Examples.Motor</strong> where the
+heating of an electrical motor is modelled, see the following screen shot
+of this example:
+</p>
+
+<p>
+<img src=\"modelica://Modelica/Resources/Images/Thermal/HeatTransfer/driveWithHeatTransfer.png\" alt=\"driveWithHeatTransfer\">
+</p>
+
+<p>
+The <strong>filled</strong> and <strong>non-filled red squares</strong> at the left and
+right side of a component represent <strong>thermal ports</strong> (connector HeatPort).
+Drawing a line between such squares means that they are thermally connected.
+The variables of a HeatPort connector are the temperature <strong>T</strong> at the port
+and the heat flow rate <strong>Q_flow</strong> flowing into the component (if Q_flow is positive,
+the heat flows into the element, otherwise it flows out of the element):
+</p>
+<pre>   Modelica.SIunits.Temperature  T  \"absolute temperature at port in Kelvin\";
+   Modelica.SIunits.HeatFlowRate Q_flow  \"flow rate at the port in Watt\";
+</pre>
+<p>
+Note, that all temperatures of this package, including initial conditions,
+are given in Kelvin. For convenience, in subpackages <strong>HeatTransfer.Celsius</strong>,
+ <strong>HeatTransfer.Fahrenheit</strong> and <strong>HeatTransfer.Rankine</strong> components are provided such that source and
+sensor information is available in degree Celsius, degree Fahrenheit, or degree Rankine,
+respectively. Additionally, in package <strong>SIunits.Conversions</strong> conversion
+functions between the units Kelvin and Celsius, Fahrenheit, Rankine are
+provided. These functions may be used in the following way:
+</p>
+<pre>  <strong>import</strong> SI=Modelica.SIunits;
```

Comment:
Renamed imports are no longer endorsed. Please change to `import Modelica.SIunits;`

Reply:
OK, I will take care of that in the HeatTransfer package.

Reply:
I am just about to create a general PR targeting all wildcardd imports.

Reply:
OK then: I will discard my branch...